### PR TITLE
feat: rounded design for components

### PR DESCRIPTION
**Pull request by mattjoke**
* added rounded design for buttons, dialogs, tables...
* updated _styles.scss, to include the changes

### CODE REVIEW

File: apps/publications/src/app/components/authors-list/authors-list.component.html [hunk 1]
```diff
@@ -82,6 +82,6 @@
   {{'SHARED_LIB.UI.ALERTS.NO_FILTER_RESULTS_ALERT' | translate}}
 </perun-web-apps-alert>
 
-<perun-web-apps-alert *ngIf="authors.length === 0" alert_type="warn">
+<perun-web-apps-alert *ngIf="authors.length !== 0" alert_type="warn">
```
Comment (bodnara) [inline apps/publications/src/app/components/authors-list/authors-list.component.html:85]: Seems unrelated.

**Review by bodnara — APPROVED** [review]
Very well done with the changes, I think it should cover all cases.

**Review by xflord — APPROVED** [review]
looks good
